lua/tardis/interiors/default.lua

```diff
@@ -223,6 +223,11 @@ T.Exterior = {
     Parts = {
         door = true
     },
+    Light = {
+        enabled = true,
```

Comment:
You can get rid of the enabled/pos here as they are inherited from the base interior